disasm/isa_parser.cc

```diff
@@ -314,6 +320,31 @@ isa_parser_t::isa_parser_t(const char* str, const char *priv)
       extension_table[EXT_ZICFILP] = true;
     } else if (ext_str == "zicfiss") {
       extension_table[EXT_ZICFISS] = true;
+    } else if (ext_str.substr(0, 3) == "zvl") {
+      try {
+        vlen = std::stol(ext_str.substr(3, ext_str.size() - 4));
+      } catch (std::logic_error& e) {
+        vlen = 0;
+      }
+      if ((vlen & (vlen - 1)) != 0 || vlen < 32)
+        bad_isa_string(str, ("Invalid Zvl string: " + ext_str).c_str());
+    } else if (ext_str.substr(0, 3) == "zve") {
+      try {
+        elen = std::stol(ext_str.substr(3, ext_str.size() - 4));
+      } catch (std::logic_error& e) {
+        elen = 0;
+      }
+      if (ext_str.substr(5) == "d") {
+        zvd = true; zvf = true;
+      } else if (ext_str.substr(5) == "f") {
+        zvd = false; zvf = true;
```

Comment:
Similarly, Zve64f doesn't imply Zve64d is absent, so these could become `|=` instead of `=`.